ansible/roles/ufw/tasks/proxy_ufw.yml

```diff
@@ -0,0 +1,35 @@
+# Rules for monoliths
+- name: allow web traffic
+  ufw:
+    rule: allow
+    direction: in
+    port: "{{ item }}"
+    proto: tcp
+  with_items:
+    - 443  # https
+    - 80   # http
+  when: is_monolith
+
+- name: allow ssh traffic
```

Comment:
are there not multiple interfaces, one for the VPN and one public? can we not have ufw be more permissive on the VPN interface than on the public one?